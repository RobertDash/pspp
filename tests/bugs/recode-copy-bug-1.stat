TITLE 'Test for regression of recode COPY bug'

DATA LIST LIST
 /A (A1)
 B (A1).

BEGIN DATA
1     2
2     3
3     4
END DATA.

** Clearly, the else=copy is superfluous here
RECODE A ("1"="3") ("3"="1") (ELSE=COPY).
EXECUTE.
LIST.

