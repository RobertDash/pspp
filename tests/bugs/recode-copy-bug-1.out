1.1 DATA LIST.  Reading free-form data from the command file.
+--------+------+
|Variable|Format|
#========#======#
|A       |A1    |
|B       |A1    |
+--------+------+
A B
- -
3 2 
2 3 
1 4 
